server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/QuotaSerializer.scala

```diff
@@ -42,6 +43,23 @@ object QuotaSerializer {
   private implicit val stateWrites: Writes[UuidState] = Json.valueWrites[UuidState]
 
   private implicit val resourceTypeWrite: Writes[ResourceType] = resourceType => JsString(resourceType.asString())
+  private implicit val scopeReads: Reads[Scope] = {
+    case JsString("account") => json.JsSuccess(AccountScope)
+    case _ => JsError(s"Expecting a JsString to represent a scope property")
```

Comment:
We caan refine error handling here.

If I pass a wrong JsString it returns me "please send me a JsString" while it should say "Oops, I do not manage this value"

```suggestion
    case JsString("account") => json.JsSuccess(AccountScope)
    case JsTring(any) => JsError(s"Unexpected value $any, only 'account' is managed")
    case _ => JsError(s"Expecting a JsString to represent a scope property")
```